.gitignore

```diff
@@ -79,3 +79,5 @@ tags
 .vscode/*
 .history
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
+#IDE (GoLand) specific
+.idea/
```

Comment:
nit, please add a new line at the end of the file.

Reply:
Done